bin/embedding-sdk/generate-sdk-package-files.js

```diff
@@ -67,14 +67,18 @@ function generateSdkPackage() {
   );
 }
 
-function generateLicenseFile() {
-  const licenseContent = fs.readFileSync(
-    path.resolve(`./enterprise/LICENSE.txt`),
+/**
+ * @param {string} source
+ * @param {string} target
+ */
+function copyFileToOutput(source, target = source) {
+  const fileContent = fs.readFileSync(
+    path.resolve(`./enterprise/${source}`),
     "utf-8",
   );
 
   fs.writeFileSync(
-    path.resolve(path.join(SDK_DIST_DIR), "LICENSE.txt"),
+    path.resolve(path.join(SDK_DIST_DIR), target),
     licenseContent,
```

Comment:
Ahh, you gotta change this one to 
```suggestion
    fileContent,
```
too. The script isn't runnable right now.